File: auto-docs/executables/nodejs/error_bar_horizontal.js
var plotly = require('plotly')('TestBot', 'r1neazxo9w')

var data = [
  {
    x: [1, 2, 3, 4], 
    y: [2, 1, 3, 4], 
    error_x: {
      type: "percent", 
      value: 10
    }, 
    type: "scatter"
  }
];

var graph_options = {filename: "error-bar-horizontal", fileopt: "overwrite", auto_open: "false"}
plotly.plot(data, graph_options, function (err, msg) {
    console.log(msg);
});

File: auto-docs/executables/python/error_bar_horizontal.py
# Learn about API authentication here: {BASE_URL}/python/getting-started
# Find your api_key here: {BASE_URL}/settings/api

import plotly.plotly as py
from plotly.graph_objs import *

py.sign_in('TestBot', 'r1neazxo9w')
data = Data([
    Scatter(
        x=[1, 2, 3, 4],
        y=[2, 1, 3, 4],
        error_x=ErrorX(
            type='percent',
            value=10
        )
    )
])
plot_url = py.plot(data, filename='error-bar-horizontal', auto_open=False)
